.github/draft_release_notes.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+
+# this script assumes the GITHUB_TOKEN and PREVIOUS_TAG environment variables have been set;
+# it produces a 'Changes.md' file as its final output;
+# the file 'last-300-prs-with-release-note.txt' that is produces is intermediate data; it is not
+# pruned for now to assist development of the release notes process (we are still curating all this)
+
+sudo apt-get -y install jq
+sudo apt-get -y install wget
```

Comment:
I might be missing something with regards to how it is handled in GitHub Actions, but before installing, it is usually a good idea to update the APT meta data beforehand:

```suggestion
sudo apt-get update
sudo apt-get install -y jq wget
```

Reply:
part of next push ... also added -y to the update invocation